WordPress/Classes/Utility/ImageURLs/NSString+PhotonURL.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NSString+PhotonURL.swift
+//  WordPress
+//
+//  Created by Andrew McKnight on 11/22/16.
+//  Copyright © 2016 WordPress. All rights reserved.
+//
+
+import Foundation
+
+extension NSString
+{
+    // Possible matches are "i0.wp.com", "i1.wp.com" & "i2.wp.com" -> https://developer.wordpress.com/docs/photon/
+    func isPhotonURL() -> Bool {
+        return self.containsString(".\(WPImageURLHelper.wordpressURLBase)")
```

Comment:
I realize this is mostly a port from existing code, but maybe this check could be improved?  Currently it will match any string containing ".wp.com" yah?  So it would be true for any ordinary text that happened to include that string (like this comment) or any other .wp.com subdomain.